include/materials/tensor.tcc

```diff
@@ -0,0 +1,226 @@
+//! Compute mean stress p
+inline double mpm::tensor::compute_mean_p(
```

Comment:
This should be const everything

Reply:
Almost all functions should be `const` everything

Reply:
Alright, I will `const` everything.